command.c

```diff
@@ -1592,36 +1592,24 @@ static void perform_loadhigh(const char *arg)
   __dpmi_int(0x21, &r);
   }
 
-static void perform_loadfix(const char *arg)
-  {
-  const int numblocks = 16;
-  int orig_strat, orig_umblink, allocated, ii;
-  unsigned short allocations[numblocks], allocation, to_64kib, max, size;
+
+#define LOADFIX_NUMBLOCKS 16
+const int loadfix_numblocks = LOADFIX_NUMBLOCKS;
+unsigned short loadfix_allocations[LOADFIX_NUMBLOCKS];
+int loadfix_ii;
+unsigned loadfix_is_v;
```

Comment:
Forgotten "static"s everywhere.
Btw, why the need for global
loadfix_is_v?

Reply:
Fair on the `static`. The `loadfix_is_v` is needed because, again, if we do explicit LOADFIX then the first call to `loadfix_exit` happens in `perform_external_cmd`, which passes an unconditional zero for the `is_v` parameter. So combine it with what `loadfix_init` (the first call to it) wrote there.

Reply:
So the param to perform_external_cmd()
can eliminate this.